addons/product/models/product_template.py

```diff
@@ -513,6 +513,19 @@ def _name_search(self, name, args=None, operator='ilike', limit=100, name_get_ui
             '', args=[('id', 'in', list(searched_ids))],
             operator='ilike', limit=limit, name_get_uid=name_get_uid)
 
+    def action_open_label_layout(self):
+        view = self.env.ref('product.product_label_layout_form')
+        return {
+            'name': _('Choose Labels Layout'),
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'product.label.layout',
+            'views': [(view.id, 'form')],
+            'view_id': view.id,
```

Comment:
`view_mode` and `view_id` are only used to generate the `views`, providing all of them should be redundant.